src/components/Home/Home.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+
+class Home extends Component {
+  constructor() {
```

Comment:
You don't need to add a constructor and invoke super if you don't intend to do anything in the constructor. You could just remove the constructor and leave the `state` part intact, and everything will work as usual.

Reply:
Thank you for your suggestion. I already fixed and pushed it.